crates/codegen/parser/runtime/src/napi_interface/cst.rs

```diff
@@ -119,28 +119,29 @@ impl TokenNode {
 }
 
 pub trait ToJS {
```

Comment:
My understanding is that `ToNapiValue` is already implemented for `Rc<>` .. do we still need `ToJS` here?
Otherwise, I wonder if we are able to make it private.